### ISSUE TITLE
dvc run doesn't work well if file is updated and not recrated

### ISSUE THREAD

**Issue by ophiry** [dataversioncontrol/dvc#61]
using a dummy script that redirects it's output to a data file, rerunning dvc run failed, unless I deleted the file and recreated it

**Comment by dmpetrov** [comment]
@ophiry, thank you for using dvc!

Could you please share more details? Have you specified --stdout for the output? Standard redirection `>` does not work in DVC, --stdout is needed.

It should go like this:
```bash
$ dvc run python saysomething.py param1 param2 --stdout data/out.txt
```

**Comment by ophiry** [comment]
I used a shell script that used redirection inside it 

the script:
```
#!/bin/sh
date > data/date.txt

```
the command

```
$ dvc run ./1.sh                                                                                                                                                                                                                        20:27:14
Debug. [dvc-git] Getting file statuses. Command: git status --porcelain
Debug. [dvc-git] Getting file statuses. Success.
Debug. Run command with args: ./1.sh. Data items from args: . stdout=None, stderr=None, shell=False
Debug. [Repository change] Exec command: ./1.sh. stdout=None, stderr=None, shell=False
Debug. [dvc-git] Getting file statuses. Command: git status --porcelain
Debug. [dvc-git] Getting file statuses. Success.
Debug. [dvc-git] Commit all changes. Commands: git add --all && git status --porcelain && git commit -m
Error. Git command error (git commit -m DVC run: ./1.sh):

```

**Comment by dmpetrov** [comment]
Yeah, I have an assumption that file should not be updated. And this is a too strong assumption.
Thank you for reporting! I'm going to fix that. The package will be updated today or tomorrow.

A workaround:
```bash
$ dvc remove data/date.txt
$ dvc run bash 1.sh
```

**Comment by dmpetrov** [comment]
Fixed. A new DVC version will be uploaded soon.

**Comment by dmpetrov** [comment]
The update is available - version 0.8.4.
